c_sharp_json_diff/IntDiffProcessor.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace c_sharp_json_diff
+{
+    class IntDiffProcessor : DiffProcessor
+    {
+
+        public override Dictionary<string, object> PerformDiffProcess(JToken left, JToken right)
+        {
+            string leftValueAsString = left.ToString();
+            string rightValueAsString = right.ToString();
+            NumericDiff diff = new NumericDiff();
+
+            if (int.TryParse(leftValueAsString, out int leftValueAsInt) && int.TryParse(rightValueAsString, out int rightValueAsInt))
+            {
+                diff.Subtraction = leftValueAsInt - rightValueAsInt;
```

Comment:
Change the order to right minus left and right / left. This is the usual way that deltas/diffs are calculated.